product_docs/docs/eprs/7/installing/uninstalling.mdx

```diff
@@ -18,9 +19,9 @@ If you installed Replication Server using the Replication Server installer progr
 
 ## Uninstalling a RPM package installation
 
-If you installed Replication Server from the RPM package, uninstall it using the Yum package manager. See Section [Uninstalling a RPM package installation](06_uninstalling_xdb_replication_server/#uninstalling-a-rpm-package-installation) for information.
+If you installed Replication Server from the RPM package, uninstall it using the yum package manager. 
 
-If you installed Replication Server from the RPM package, you can uninstall any Replication Server component by invoking the `yum remove package_name` command as the root account where `package_name` is any Replication Server component RPM package as listed in the table in Section [Installing the Replication Server RPM Package](03_installing_rpm_package/#installing_rpm_package).
+If you installed Replication Server from the RPM package, you can uninstall any Replication Server component by invoking the `yum remove package_name` command as the root account where `package_name` is any Replication Server component RPM package is `edb-xdb-console`, `edb-xdb-publisher`, or `edb-xdb-subscriber`.
```

Comment:
Wording seems incorrect.  See "is" before the list of packages

Reply:
Trying a different format.